modules/nodes-nodes-rebooting-gracefully.adoc

```diff
@@ -7,6 +7,13 @@
 
 Before rebooting a node, it is recommended to backup etcd data to avoid any data loss on the node.
 
+[NOTE]
+====
+For  Single Node OpenShift (SNO) clusters that require users to perform the 'oc login' command rather than having the certificates in kubeconfig file to manage the cluster, after cordoning and draining the node, the 'oc adm' commands might not be available because the `openshift-oauth-apiserver` pod is not running because of the cordon. You can use SSH to access the nodes as indicated in the following procedure.
```

Comment:
Tried to get some of the modifiers closer together, but would be ideal to break up that first sentence if you can figure out a good way to do so.

```suggestion
For Single Node OpenShift (SNO) clusters that require users to perform the `oc login` command rather than having the certificates in `kubeconfig` file to manage the cluster, the `oc adm` commands might not be available after cordoning and draining the node because the `openshift-oauth-apiserver` pod is not running due to the cordon. You can use SSH to access the nodes as indicated in the following procedure.
```

Reply:
Good catch on the back ticks!